### ISSUE TITLE
Convert Hits from a Search to Fasta files

### ISSUE THREAD

**Issue by juliacpowell1999** [soedinglab/MMseqs2#492]
Hi I am new to MMseqs2 (and the coding world in general). What I want to do I am sure is very simple I just cannot seem to figure out how to do it.

I ran a query protein sequence against the UniProtKB/Swiss-Prot database. The resultant ResultDB showed 20 hits. I want to extract those sequences corresponding to the 20 hits to do a MSA using ClustalO. Is there a way to obtain the actual sequences corresponding to the hits from a search query and convert them to fasta format?

I tried using the creattsv function to convert the ResultDB to a tsv file, but because there is no header information the convert2fasta function did not work.

Any help or suggestions would be appreciated. 

Thanks!

**Comment by martin-steinegger** [comment]
@juliacpowell1999 Yes you can get the target sequences by adding `tseq` to the `--format-output` options. 
For example: 
```
easy-search query target result tmp --format-output query,target,tseq
```

**Comment by juliacpowell1999** [comment]
> @juliacpowell1999 Yes you can get the target sequences by adding `tseq` to the `--format-output` options. For example:
> 
> ```
> easy-search query target result tmp --format-output query,target,tseq
> ```

Thank you! I now need to convert those sequences in the file to fasta format, how would I do that. I tried to using the convert2fasta command but an error keeps occuring because the file does not contain header information.

**Comment by martin-steinegger** [comment]
Closing this one due to inactivity — it was answered above and we haven't heard back. If it's still relevant, please reopen with your MMseqs2 version, the exact command, and the full output, and we'll be happy to help. Thanks!
